### ISSUE TITLE
Extend analysis table without overwriting

### ISSUE THREAD

**Issue by ml-evs** [force-h2020/force-wfmanager#267]
It would be nice if some parameters could be marked with something like "does not affect results", e.g. something like the number of trials/iterations an MCO performs. If a workflow is unchanged from its previous run, apart from these parameters, the results table should be updated rather than overwritten.

**Comment by flongford** [comment]
I agree, this would be a nice addition, as well as being able to append new data to the results table. Also this is required for the Unilever test case, since the simulation experiments are too costly to throw away the results from a run or perform the same calculation twice.

However, at the moment the initial parameters for each MCO iteration are generated in each plugin's `BaseMCO`, whereas the wfmanager `analysis_model` only stores the final optimised parameters and KPIs. In order to improve the performance therefore the assessment would need to be done before execution, and so handled in the `force_bdss`.

Also, if one of the data models in the workflow uses a random number generator, then the seed also needs to be specified in order for the optimised variables to be the same for a given set of initial variables. Therefore running the calculation again with the same number of trials/iterations may produce new results.
